app/models/source.rb

```diff
@@ -37,7 +37,18 @@ def self.datatable_scope
   end
 
   def display_name
-    description
+    "#{self.description} - (#{source_type_description})"
```

Comment:
I think the display_name might be used in other places as well so we should make sure that we want to display the source type there as well.

Reply:
Yeah, I was trying to figure that out - we have both `description` and `display_name`, we should probably take a quick inventory of what's used where. I did check the main evidence item view and it did not change it there (which I think is the correct behavior).

Reply:
Yeah, the `display_name` seems to only be used in the state params as far as I can tell.

Reply:
Yeah, I concur, I think that's fine for now. In general for v2 we probably want to rethink the whole name/description/display_name thing we've got going on. Its not totally clear which one to use when and where. (That's totally my fault).